articles/portal/ptl-ref-portal-api.md

```diff
@@ -212,593 +212,45 @@ Returns a response with a list of up to 10 VMs
 ```
 
 ## GET /api/my_calls
-
+#### With the move to the ITSM tool this functionality is no longer available.
```

Comment:
There's no documentation for the ITSM tool just yet, but we should remember to update this page with a link once there is.